majasdarbs/src/io/codelex/arithmetic/practice/Exercise8.java

```diff
@@ -0,0 +1,52 @@
+package io.codelex.arithmetic.practice;
+
+import java.util.Scanner;
+
+public class Exercise8 {
+
+    static double basePay = 0;
```

Comment:
Ja šie mainīgie tiek ievadīti, tad labāk viņus arī pašā for ciklā un padot `calculatePay()` metodei kā parametrus, nevis definēt kā klases mainīgos.